pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -34,7 +34,7 @@ const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
 	ControllerRuntimeVersion = "v0.14.4"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
-	ControllerToolsVersion = "v0.11.3"
+	ControllerToolsVersion = "v0.14.0"
```

Comment:
We should not change go/v3. It is deprecated and will be soon no longer supported
More info: https://github.com/kubernetes-sigs/kubebuilder/discussions/3622